app/assets/stylesheets/pages/_events_show.scss

```diff
@@ -0,0 +1,55 @@
+
+.logo {
+text-align: center;
+font-weight: 300;
+font-size: 35px;
+}
+
+.title1 {
+  text-align: center;
+  top: 40px;
+  font-weight: bolder;
+  margin-top: 100px;
+  font-size: 30px;
+}
+
+.button1 {
```

Comment:
be more specific. No numbering, think visually. What kind of button is this? big, small, green, orange, whatever